### PR TITLE
Retain original layer name in the compressed model

### PR DESCRIPTION
**Pull request by l-bat**

### CODE REVIEW

File: beta/nncf/tensorflow/layers/wrapper.py [hunk 1]
```diff
@@ -45,11 +45,14 @@ def __init__(self, layer, **kwargs):
                     input=layer.__class__.__name__))
 
         if 'name' not in kwargs:
-            kwargs['name'] = '{}_{}'.format('nncf_wrapper', layer.name)
+            kwargs['name'] = layer.name
 
         super().__init__(layer, **kwargs)
         self._track_trackable(layer, name='layer')
 
+        if 'name' not in kwargs:
```
Comment (alexsu52) [inline beta/nncf/tensorflow/layers/wrapper.py:53]: It looks like we can leave self.layer.name. Could you check it?

Comment (l-bat) [reply beta/nncf/tensorflow/layers/wrapper.py]: I removed these changes and tests passed

File: beta/tests/tensorflow/data/ac_configs/retinanet_int8_w_sym_t_a_sym_t.yml [hunk 2]
```diff
@@ -6,17 +6,17 @@ models:
...
-            - retinanet/nncf_wrapper_class-predict/BiasAdd_4/Add
-            - retinanet/nncf_wrapper_class-predict/BiasAdd_3/Add
-            - retinanet/nncf_wrapper_class-predict/BiasAdd_2/Add
-            - retinanet/nncf_wrapper_class-predict/BiasAdd_1/Add
-            - retinanet/nncf_wrapper_class-predict/BiasAdd/Add
+            - retinanet/class-predict/BiasAdd_4/Add
+            - retinanet/class-predict/BiasAdd_3/Add
+            - retinanet/class-predict/BiasAdd_2/Add
+            - retinanet/class-predict/BiasAdd_1/Add
+            - retinanet/class-predict/BiasAdd/Add
```
Comment (pfinashx) [inline beta/tests/tensorflow/data/ac_configs/retinanet_int8_w_sym_t_a_sym_t.yml:19]: Hello! After changes we don't need retinanet_int8_w_sym_t_a_sym_t.yml  anymore. It's will be created in test on-the-fly, but for this need modification test_sota_checkpoints.py

**Review by alexsu52 — COMMENTED** [review]
Looks good to me!

File: beta/tests/tensorflow/data/ac_configs/retinanet_int8_w_sym_t_a_sym_t.yml (hunk 2 above)
Comment (l-bat) [reply beta/tests/tensorflow/data/ac_configs/retinanet_int8_w_sym_t_a_sym_t.yml]: Deleted

File: beta/tests/tensorflow/test_weekly.py [hunk 3]
```diff
@@ -0,0 +1,278 @@
...
+# │   ├── path
+# │   ├── batch
+# │   ├── configs
+# │   │     ├─── config_filename
+# │   │     │       ├── expected_accuracy
+# │   │     │       ├── absolute_tolerance_train
+# │   │     │       ├── absolute_tolerance_eval
+# │   │     │       ├── execution_arg
+# │   │     │       ├── weights
+GLOBAL_CONFIG = {
```
Comment (daniil-lyakhov) [inline beta/tests/tensorflow/test_weekly.py:38]: Minor comment: what are the reasons behind splitting configs on `sota_checkpoints_eval.json` and `GLOBAL_CONFIG`? I mean, it's look like duplication, and if one would like to change target accuracy he should change it in both at `sota_checkpoints_eval.json` and `GLOBAL_CONFIG`

**Comment by l-bat** [comment]
Build 174

File: beta/tests/tensorflow/test_weekly.py (hunk 3 above)
Comment (l-bat) [reply beta/tests/tensorflow/test_weekly.py:38]: Yes, right now it's duplication.
We are planning to add configs for cifar100 to `GLOBAL_CONFIG` and run it  weekly on CI.
For object detection and segmentation models, we need to specify FP32 weights, set `absolute_tolerance_train` and `absolute_tolerance_test`.

**Review by alexsu52 — APPROVED** [review]

**Review by daniil-lyakhov — APPROVED** [review]

**Review by pfinashx — APPROVED** [review]

**Pull request merged**